tracks/csharp/exercises/sum-of-multiples/README.md

```diff
@@ -0,0 +1,60 @@
+### Reasonable solutions
+
+```csharp
+using System.Linq;
+using System.Collections.Generic;
+
+public static class SumOfMultiples
+{
+    public static int Sum(IEnumerable<int> inputs, int max)
+        => inputs.SelectMany(
+            input => Enumerable.Range(1, (max - 1) / input).Select(numTimes => numTimes * input)
+            , (numtimes, multiple) => multiple).Distinct().Sum();
+}
+```
+
+If the student is not yet ready for / comfortable with full-on Linq then something like the following would be appropriate:
+
+```csharp
+using System.Linq;
+using System.Collections.Generic;
+
+public static class SumOfMultiples
+{
+    public static int Sum(IEnumerable<int> inputs, int max)
+    {
+        var hs = new HashSet<int>();
+        foreach (var input in inputs)
+        {
+            for (int ii = (max - 1) / input * input; ii > 0 ; ii -= input)
+            {
+                hs.Add(ii);
+            }
+        }
+
+        return hs.Sum();
+    }
+ }
+ ```
+
+### Common suggestions
+
+- There are dozens of equally valid linq constructions that can be used.
+- expression bodied members are not to everybody's taste and should be optional
```

Comment:
Maybe you could use the exact same line that we use in previous mentor notes (e.g. in [leap](https://github.com/exercism/mentors/tree/master/tracks/csharp/exercises/leap))? This would also mean moving it to the talking points section.

Reply:
done